src/test/java/com/razorpay/StakeholderClientTest.java

```diff
@@ -200,4 +200,53 @@ private String getResponseCollection(){
                 "  \"count\": 1\n" +
                 "}";
     }
+
+    @Test
+    public void testuploadStakeholderDoc() throws RazorpayException {
+        JSONObject request = new JSONObject();
+        request.put("files","/Users/your_name/Downloads/sample_uploaded.pdf");
```

Comment:
use relative URL

Reply:
I placed the same URL as it is in the document.

Reply:
https://github.com/razorpay/razorpay-java/blob/530334834fab1783cb3abd7e26e87a56681d2383/src/test/java/com/razorpay/StakeholderClientTest.java#L208